schema/2.0.0/seurat_encoding.md

```diff
@@ -0,0 +1,226 @@
+
+# Schema - Seurat encoding
+
+Contact: acarr@chanzuckerberg.com
+
+Document Status: _In progress_
+
+Schema version: 2.0.0
+
+
+## Overview
+
+All data submitted to the [cellxgene Data Portal](https://cellxgene.cziscience.com/) is automatically converted to a Seurat V3 object that can be loaded by the R package [Seurat](https://satijalab.org/seurat/).
+
+This document describes the Seurat encoding for the converted data. <u><strong>Readers should be familiar with the [schema](.schema.md).</strong></u>
+
+## Encoding
+
+The Seurat V3 object is stored using the [native serialization RDS format](https://cran.r-project.org/doc/manuals/r-release/R-ints.html#Serialization-Formats). The [serialization interfaces](https://stat.ethz.ch/R-manual/R-devel/library/base/html/readRDS.html) allow the object to be serialized to and restored from disk. 
+
+A `local.rds` file downloaded from the cellxgene Data Portal can be read into an R session with the following code.
+
+```r
+seurat_object <- readRDS("local.rds")
+```
+
+The following sections describe the individual components of a dataset as encoded in the Seurat object.
+
+### Data matrix
+
+Matrix data is stored in the slot `assays` under the element `RNA`.
+
+#### Raw data
+
+<table><tbody>
+    <tr>
+      <th>Slot</th>
+      <td><code>counts</code></td>
+    </tr>
+    <tr>
+      <th>Access</th>
+      <td><code>seurat_object@assays$RNA@counts</code></td>
```

Comment:
We have ATAC and mC data in the portal. Should those be stored in `$RNA` or another slot?

Reply:
Yeah I though about this. Functionally they will work as intended even if stored in `$RNA`, but obviously it's not the best naming. As far as I understand `@assays` was designed for multi-assay experiments that share metadata (e.g. CITE-seq). At some point we do have to implement a proper conversion for these kind of data. 

As for now, only the naming will not be correct. But functionally it will work. I don't think changing the names is worth investing on at the moment, but rather just documenting it.

Thoughts?

Reply:
This makes sense to me. I was wondering if there was an obvious change. Perhaps this is something we could flag for Rahul's team when they review.